### ISSUE TITLE
Multivariate demo reconstruction window error

### ISSUE THREAD

**Issue by yaseryacoob** [sintel-dev/Orion#298]
I tried to test the multivariate tutorial, the reconstruction part runs fine for windows of 100, but when I changed the hyperparameter values to 200 I get an error, I also tried to make the change in the lstm_autoencoder.json and got same error. See both below.

What am I missing?

------Hyper parameter changed to, and the error comes from keras
https://github.com/sintel-dev/Orion/blob/master/tutorials/Multivariate_Input.ipynb
```
hyperparameters = {
    "mlprimitives.custom.timeseries_preprocessing.rolling_window_sequences#1": {
        "window_size": 200,
        "target_column": 0
    },
    'keras.Sequential.LSTMSeq2Seq#1': {
        'epochs': 1,
        'verbose': True,
        'input_shape': [200, 25],
        'target_shape': [200, 1],
    }
}
```
---------------- same error when I also changed lstm_autoencoder.json to 200 
```
(orionGPU3) yaser@yaserubuntu:/media/ydrive/time_series/Orion$ python Multivariate_Input.py
/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/sklearn/utils/validation.py:37: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  LARGE_SPARSE_SUPPORTED = LooseVersion(scipy_version) >= '0.14.0'
(10149, 26)
Using TensorFlow backend.
YYprocessing lstm_autoencoder
Exception caught fitting MLBlock keras.Sequential.LSTMSeq2Seq#1
Traceback (most recent call last):
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/mlblocks/mlpipeline.py", line 549, in _fit_block
    block.fit(**fit_args)
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/mlblocks/mlblock.py", line 302, in fit
    getattr(self.instance, self.fit_method)(**fit_kwargs)
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/mlprimitives/adapters/keras.py", line 99, in fit
    self.model = self._build_model(**kwargs)
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/mlprimitives/adapters/keras.py", line 54, in _build_model
    model.add(built_layer)
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/keras/engine/sequential.py", line 182, in add
    output_tensor = layer(self.outputs[0])
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/keras/engine/base_layer.py", line 506, in __call__
    output_shape = self.compute_output_shape(input_shape)
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/keras/layers/core.py", line 399, in compute_output_shape
    input_shape[1:], self.target_shape)
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/keras/layers/core.py", line 387, in _fix_unknown_dimension
    raise ValueError(msg)
ValueError: total size of new array must be unchanged
Traceback (most recent call last):
  File "Multivariate_Input.py", line 86, in <module>
    orion.fit(data)
  File "/media/ydrive/time_series/Orion/orion/core.py", line 86, in fit
    self._mlpipeline.fit(data, **kwargs)
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/mlblocks/mlpipeline.py", line 649, in fit
    self._fit_block(block, block_name, context)
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/mlblocks/mlpipeline.py", line 549, in _fit_block
    block.fit(**fit_args)
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/mlblocks/mlblock.py", line 302, in fit
    getattr(self.instance, self.fit_method)(**fit_kwargs)
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/mlprimitives/adapters/keras.py", line 99, in fit
    self.model = self._build_model(**kwargs)
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/mlprimitives/adapters/keras.py", line 54, in _build_model
    model.add(built_layer)
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/keras/engine/sequential.py", line 182, in add
    output_tensor = layer(self.outputs[0])
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/keras/engine/base_layer.py", line 506, in __call__
    output_shape = self.compute_output_shape(input_shape)
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/keras/layers/core.py", line 399, in compute_output_shape
    input_shape[1:], self.target_shape)
  File "/home/yaser/miniconda3/envs/orionGPU3/lib/python3.7/site-packages/keras/layers/core.py", line 387, in _fix_unknown_dimension
    raise ValueError(msg)
ValueError: total size of new array must be unchanged
(orionGPU3) yaser@yaserubuntu:/media/ydrive/time_series/Orion$
```

**Comment by sarahmish** [comment]
Thank you @yaseryacoob for pointing this issue! 

There is an incompatibility in building the layers of the autoencoders, working on this right now!

**Comment by sarahmish** [comment]
Fixed in PR #300
